mule-apikit-metadata/src/main/java/org/mule/module/apikit/metadata/ApplicationModelWrapper.java

```diff
@@ -67,57 +69,48 @@ private void findAndProcessFlows() {
     final Map<String, RamlCoordinate> flowMappingCoordinates = createCoordinatesForMappingFlows(flows, coordsFactory);
 
     // Merging both results
-    metadataFlows = Stream.of(conventionCoordinates, flowMappingCoordinates)
-        .flatMap(map -> map.entrySet().stream())
-        .collect(toMap(Map.Entry::getKey, Map.Entry::getValue));
+    metadataFlows = merge(conventionCoordinates, flowMappingCoordinates);
+  }
+
+  private static <K, V> Map<K, V> merge(Map<K, V> a, Map<K, V> b) {
```

Comment:
Maybe you could move this function to a Utils class. And it can receive `Map<K,V>...` and/or an array as a parameter, to allow merging multiple Maps.